specs/Print.md

```diff
@@ -0,0 +1,473 @@
+
+Print API
+===
+
+# Background
+
+Developers have requested the ability to programmatically print the current page optionally using a print dialog in WebView2. Currently the end user can open a print preview dialog by pressing "Ctrl+ P" or selecting "Print" from the context menu in a document or can open the system print dialog by pressing
```

Comment:
Editing note: Please use line returns at ~100 columns; not all viewers support wrapping (including GitHub's)